### PR TITLE
fix assigned but unused variable warnings

### PR DESCRIPTION
**Pull request by walf443**
Testing with -w option output following warnings:

```
$ bundle exec  rspec -w
./spec/support/active_record.rb:151: warning: assigned but unused variable - primary_key
./spec/support/active_record.rb:4: warning: already initialized constant ROOT
./spec/support/active_record.rb:4: warning: previous definition of ROOT was here
./spec/support/active_record.rb:6: warning: already initialized constant DB_CONFIG
./spec/support/active_record.rb:6: warning: previous definition of DB_CONFIG was here
```

This warning means primary_key is considered just variable not "#primary_key=" method.
So calling it with `self.` is required.

### CODE REVIEW

**Comment by brendon** [comment]
Thanks @walf443, good catch. And thank you again for all your work on this gem! I do very much appreciate it :)

It looks like we're loading this file twice (at least). Perhaps we should deal with that (RE the reinitialised constants) rather than conditionally set the constant? We could define those constants in the rspec support harness instead?

File: spec/support/active_record.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
 require 'active_record'
 require 'logger'
 
-ROOT = File.join(File.dirname(__FILE__), '..') unless defined?(ROOT)
```
Comment (walf443) [inline spec/support/active_record.rb]: ROOT seems to be not used.

**Comment by brendon** [comment]
One failing test for some reason:

```
Duck
609  a rearrangement
610    with max value
611      
...
614      
615        should eq [201, 204, 207, 208, 209, 210, 211, 212, 213, 214, 215, 216, 217, 218, 219, 220, 221, 222, 223, 224, ... 382, 383, 384, 385, 386, 387, 388, 389, 390, 391, 392, 393, 394, 395, 396, 397, 398, 399, 400, 202] (FAILED - 1)
616    with min value
```

We sometimes get an intermittent test failure with Postgres. I've never been able to figure it out but it seems like a timing issue. I've rebuilt the test, so hopefully it passes.

**Comment by walf443** [comment]
@brendon Test passed!!

**Comment by brendon** [comment]
Nice! One day I'd like to get to the bottom of that issue, but for today we can carry on :D

**Pull request merged**